<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Igor Malinovskiy</author>
 <class>demoClass</class>
 <widget class="QMainWindow" name="demoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>2</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Redis Desktop Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/rdm.qrc">
    <normaloff>:/images/redisBigIcon.png</normaloff>:/images/redisBigIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: open-sans, ubuntu, sans-serif;</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="connectionsTreeLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="ConnectionsTree::TreeView" name="serversTreeView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>330</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>800</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>330</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family: monospaced;</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>300</number>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QToolButton" name="pbImportConnections">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>190</width>
              <height>35</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>145</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>open-sans,ubuntu,sans-serif</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Import / Export</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/rdm.qrc">
              <normaloff>:/images/menu.png</normaloff>:/images/menu.png</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="pbAddServer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>140</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>open-sans,ubuntu,sans-serif</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">padding-left: 50px;

</string>
            </property>
            <property name="text">
             <string>Connect to Redis Server</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/rdm.qrc">
              <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>620</width>
         <height>300</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>620</width>
         <height>300</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="handleWidth">
        <number>1</number>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="qmlParent" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>400</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>700</width>
          <height>450</height>
         </size>
        </property>
       </widget>
       <widget class="ConsoleTabs" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>580</width>
          <height>35</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="usesScrollButtons">
         <bool>true</bool>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConnectionsTree::TreeView</class>
   <extends>QTreeView</extends>
   <header>connections-tree/treeview.h</header>
  </customwidget>
  <customwidget>
   <class>ConsoleTabs</class>
   <extends>QTabWidget</extends>
   <header>app/widgets/consoletabs.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/rdm.qrc"/>
 </resources>
 <connections/>
</ui>
